btree/btree.c

```diff
@@ -2619,7 +2677,9 @@ static int m0_btree_put_makespace_phase(struct m0_btree_op *bop)
 			       m0_vec_count(&l_slot.s_rec.r_key.k_data.ov_vec));
 		temp_rec.r_val.ov_buf[0] =  &(curr_level->l_alloc->n_addr);
 		m0_bufvec_copy(&bop->bo_rec.r_val,
-				&temp_rec.r_val, INTERNAL_VAL_SIZE);
+			       &temp_rec.r_val, INTERNAL_VAL_SIZE);
+		/* if we want to update vcount for slot */
```

Comment:
Please remove the commented lines if not needed.

Reply:
I have added comments intentionally, as in future , we might want to change the vcount for slot in case of variable key size , so we can refer this comments